### ISSUE TITLE
Enhancement: Resizable tables

### ISSUE THREAD

**Issue by klmedia** [ondras/wwwsqldesigner#215]
Would be a nice thing if tables would be resizable. For example some tables with a lot of not so important columns takes too much space, so it could be resized, eventually with inline scrolling to view all the columns if needed? Thanks!

**Comment by ondras** [comment]
Interesting idea. Apart from the obvious "grab lower-right corner to resize" UI stuff, we would have to also provide a UI/UX-friendly way to revert back to "auto-width" mode, because users will want to go back to the previous automatic state. Perhaps a double-click on this corner can do this?

**Comment by klmedia** [comment]
I agree with your idea of reverting. Double click will make it, if we provide this info to the users how they  do it. Or an additional option when double click the table where you edit the table name and comment eventually. Or both ways. Thanks!
